CHANGELOG.rst

```diff
@@ -12,6 +12,7 @@ version develop
 -----------------
 + Use upstream ISA-L version 2.31.1 which includes patches to make 
   installation possible.
++ Make GNU/Hurd a supported platform.
```

Comment:
"supported" I think is a bit of an overstatement. Because I don't run GNU/Hurd platforms and cannot support them. 
"Enable building on GNU/Hurd platforms" is a better description.

Reply:
Proposed change implemented.